automation/issues.py

```diff
@@ -0,0 +1,53 @@
+import datetime
+import re
+from collections import defaultdict
+from typing import Generator
+
+import httpx
+from markdown_it import MarkdownIt
+from markdown_it.tree import SyntaxTreeNode
+
+
+def get_issue(issue_id: str) -> dict[str, str]:
+    """Returns the issue with the given id"""
+    url = f"https://api.github.com/repos/kjaymiller/Python-Community-News/issues/{issue_id}"  # TODO: remove hardcoded issue url
+    request = httpx.get(url)
+    return request.json()
+
+
+def get_issues(labels, since_date: datetime.datetime | None) -> list:
+    """Returns the issues filed in the last week"""
+    url = "https://api.github.com/repos/kjaymiller/Python-Community-News/issues"
+    params = {"labels": ",".join(labels), "since": since_date}
```

Comment:
Does this also support filtering by issues that are currently open? e.g. we've closed #19 in favor of a couple of other specific issues that replaced it.

Reply:
We can. Looking at this now I think I would want to change how that params field is passed to include `**kwargs` that get passed as params.